tests/config/indent_ctor_init.cfg

```diff
@@ -1,1289 +1,18 @@
-# Uncrustify 0.58
```

Comment:
I tried to diff the config with defaults for uncrustify 0.58, but there were way too many changes - it's someone's complete config with all modifications. So I kept only the options relevant to what the test case seemed to check (just one, 30955, actually references this config).

Seems pointless to keep 50 KB configs around. Like there are just minimal code snippets, there should be just minimal configs. It's much easier to see what's going on and also to create derivative configs, like I subsequently did with `indent_ctor_init_leading.cfg` here.

But all that's just IMO, if you guys prefer to keep it in its entirety, I will just remove that commit :)